modules/007-registrypackages/images/iptables/scripts/install

```diff
@@ -17,34 +17,74 @@ mkdir -p /opt/deckhouse/bin
 cp -f xtables-legacy-multi /opt/deckhouse/bin
 cp -f xtables-nft-multi /opt/deckhouse/bin
 
-kubeletChainsRegex='^:(KUBE-IPTABLES-HINT|KUBE-KUBELET-CANARY)'
-NFT_KERNEL_LIMIT=3.14 # nf_tables_ipv6, nf_tables_bridge, nf_tables_arp allow since Linux kernel >= 3.14.
-CURRENT_KERNEL_VERSION=$(uname -r | awk -F"-" '{print $1}') #version kernel view format: a.b.c-generic to a.b.c.
-IPTABLES_LEGACY_RULE=$( (/opt/deckhouse/bin/xtables-legacy-multi iptables-save || true ) 2>/dev/null | grep  -E ${kubeletChainsRegex} | wc -l )
+if [ -f /opt/deckhouse/bin/iptables ]; then
+  # if iptables symlinks already exist, we doesn't update it
+  exit 0
+fi
 
-function check_python() {
-  for pybin in python3 python2 python; do
-    if command -v "$pybin" >/dev/null 2>&1; then
-      python_binary="$pybin"
-      return 0
+function kernelNFTSupport() {
+  local configNFTablesRegex='CONFIG_NF_(TABLES|TABLES_IPV4)=(y|m)'
+  local configFiles=(
+    /boot/config-$(uname -r)
+    /proc/config.gz
+    /boot/config
+  )
+
+  for file in "${configFiles[@]}"; do
+    if [ -f "$file" ]; then
+      local matchCount
+      if [[ "$file" == *.gz ]]; then
+        matchCount=$(zgrep -Ec "$configNFTablesRegex" "$file")
+      else
+        matchCount=$(grep -Ec "$configNFTablesRegex" "$file")
+      fi
+      
+      if [[ $matchCount -eq 2 ]]; then
+        # CONFIG_NF_TABLES and CONFIG_NF_TABLES_IPV4 should be present for nft support
+        return 0
+      else
+        return 1
+      fi
     fi
   done
-  echo "Python not found, exiting..."
+
+  # If no kernel configuration files were found
   return 1
 }
 
-function isLegacyKernel() {
-  check_python
-  $python_binary -c "exit(0) if tuple(map(int, '$CURRENT_KERNEL_VERSION'.split('.'))) < tuple(map(int, '$NFT_KERNEL_LIMIT'.split('.')))  else exit(1)"
+function checkAndRemoveRule() {
+  local iptablesCommand="/opt/deckhouse/bin/xtables-nft-multi iptables"
+  local rule="-t filter -A OUTPUT -s 216.216.216.216 -d 216.216.216.216 -j ACCEPT"
+
+  # Check if the rule can be added
+  if $($iptablesCommand $rule); then
+    # Delete the rule if it was successfully added
+    $iptablesCommand -t filter -D OUTPUT -s 216.216.216.216 -d 216.216.216.216 -j ACCEPT
```

Comment:
i think we should use not real address - 127.0.0.1 for example

Reply:
idea is that  `216` (Decimal) it's `D8` (hex)

Reply:
Agree with Denys. Let,s use 127.216.216.216/32 :).

Reply:
done